parity/run.rs

```diff
@@ -141,6 +143,12 @@ pub fn execute(cmd: RunCmd) -> Result<(), String> {
 	// prepare account provider
 	let account_provider = Arc::new(try!(prepare_account_provider(&cmd.dirs, cmd.acc_conf)));
 
+	// create supervisor
+	let mut hypervisor = modules::hypervisor(::std::path::Path::new(&cmd.dirs.ipc_path()));
+	if let Some(ref stratum_cfg) = cmd.stratum {
```

Comment:
Why not put `enabled` as part of the config and check inside `modules::stratum` (as we do in other cases)?